tfe/resource_tfe_team_members_test.go

```diff
@@ -95,6 +114,12 @@ func TestAccTFETeamMembers_import(t *testing.T) {
 	})
 }
 
+func testCheckResourceAttrWithIndexesAddr(name, format string, idx *int, value string) resource.TestCheckFunc {
```

Comment:
As you defined this as a generic helper function, I doubt if it should be declared in this file? It will now be hidden away here, invisible to other people writing tests. So maybe we should place this in `provider_test.go` instead?

And I wondered why you want a `*int` here? As you don't test for it being `nil`, why not just use and `int`?

Reply:
I actually found a better/cleaner way to do this. The `*int` is just a leftover of my poor copy-pasta skill. I will remove this function altogether.